2 Business layer/CandidateEvaluator.Core/Handlers/Queries/GetCategoryHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CandidateEvaluator.Contract.Handlers;
+using CandidateEvaluator.Contract.Models;
+using CandidateEvaluator.Contract.Queries;
+using CandidateEvaluator.Contract.Services;
+using System.Threading.Tasks;
+
+namespace CandidateEvaluator.Core.Handlers.Queries
+{
+    public class GetCategoryHandler : IQueryHandler<GetCategory, Category>
+    {
+        private readonly ICategoryService _categoryService;
+
+        public GetCategoryHandler(ICategoryService categoryService)
+        {
+            _categoryService = categoryService;
```

Comment:
I believe that handlers should operate straight on repositories/context. The service layer is redundant here.